// Copyright 2020 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto3";

package feedwire;

import "components/feed/core/proto/v2/wire/action_diagnostic_info.proto";
import "components/feed/core/proto/v2/wire/client_info.proto";
import "components/feed/core/proto/v2/wire/consistency_token.proto";
import "components/feed/core/proto/v2/wire/feed_action.proto";

option optimize_for = LITE_RUNTIME;

message UploadActionsRequest {
  repeated FeedAction feed_actions = 1;
  ConsistencyToken consistency_token = 2;
  ActionDiagnosticInfo action_diagnostic_info = 5;
  ClientInfo client_info = 6;
}
